Resources/Prototypes/Reagents/toxins.yml

```diff
@@ -403,7 +403,7 @@
       - !type:HealthChange
         conditions:
         - !type:ReagentThreshold
```

Comment:
you can completely remove the condition instead of setting it to 0 since it is always true. Same with the other one below.

Reply:
Ah, ok. I wasn't sure if that'd break something in another place doing it